bcda/models/fhir/alr/alr_test.go

```diff
@@ -34,56 +37,92 @@ func TestGenerateAlr(t *testing.T) {
 	assert.NoError(t, err)
 	assert.Len(t, alrs, 1)
 
-	lastUpdated := time.Now().Round(time.Second)
-	patient, obs := alr.ToFHIR(alrs[0], lastUpdated)
-	assert.NotNil(t, patient)
-	assert.Len(t, obs, 5)
+	//lastUpdated := time.Now().Round(time.Second)
```

Comment:
remove if dropping lastUpdated, or include as part of AlrBulkFhir struct

Reply:
Will add this back in in R4 ticket.